share/completions/bzip2recover.fish

```diff
@@ -1,11 +1,9 @@
 complete -c bzip2recover -k -x -a "(
-	__fish_complete_suffix .tbz
-	__fish_complete_suffix .tbz2
+	__fish_complete_suffix .tbz .tbz2
 )
```

Comment:
If it's now only one command, just place it on a single line instead of multiple lines (same for the others, please).